docs/UserGuide.md

```diff
@@ -59,7 +59,8 @@ easily available to streamline the work of McDonald's Shift Managers.
   worker list) you wish to delete from the application. The command can be used as `worker-delete 4`.
 
 * Parameters in `[square brackets]` are optional.<br>
-  e.g. `n/NAME [r/ROLE]` can be used as `n/John Doe r/Cashier` or as `n/John Doe`.
+  e.g. `n/NAME [r/ROLE]` can be used as `n/John Doe r/Cashier` or as `n/John Doe`. `u/UNAVAILABLE_DAY [UNAVAILABLE_TIME]`
+  can be used as `u/Mon am` or as `u/Mon`.
```

Comment:
```suggestion
  e.g. `n/NAME [r/ROLE]` can be used as `n/John Doe r/Cashier` or as `n/John Doe`. `[u/UNAVAILABLE_DAY [UNAVAILABLE_TIME]]`
  can be used as ` ` (i.e. not provided), as `u/Mon` or as `u/Mon am`.
```
Perhaps this will illustrate the use of nested square brackets better.